Add some examples of reading the content of a request

quodlibetor opened the pull request.

seanmonstar commented:
Thanks! I'd probably rather keep the example simple, as it is, but I certainly think adding something to the docs about using Read would be a great thing.

quodlibetor commented:
I agree about the example, it was the first thing I wrote and then I realized that I should just improve the main docs and never went back and cleaned it up.

Not sure how you feel about the paragraph I added about Read, is it the kind of thing you were thinking of? I'm happy to clean it up or move it around.

seanmonstar commented:
This looks great to me! Thanks!
